CHANGES.txt

```diff
@@ -1,6 +1,8 @@
+Changelog moved to https://github.com/planetlabs/planet-client-python/releases
```

Comment:
this file is out of date. figured a redirect to the releases page is warranted, since that is the source of truth.